docs/source/api_ref_data.rst

```diff
@@ -14,8 +17,36 @@ torchtune.data
     AlpacaInstructTemplate
     GrammarErrorCorrectionTemplate
     SummarizeTemplate
+
+.. _chat_formats:
+
+Chat formats
+------------
+
+.. autosummary::
+    :toctree: generated/
+    :nosignatures:
+
     ChatFormat
     ChatMLFormat
     Llama2ChatFormat
     MistralChatFormat
+
+Types
+-----
+
+.. autosummary::
+    :toctree: generated/
+    :nosignatures:
+
+    Message
```

Comment:
I think sphinx breaks for Message because it does not have docstrings

Reply:
Message is fine, it broke for ``truncate``